src/_data/capability-matrix.yml

```diff
@@ -532,6 +606,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ''
+          - class: mapreduce
+            l1: 'Yes'
```

Comment:
Once you say 'No' to triggering, you can leave these blank or just say 'No' without the explanation. It is a nice explanation, though, so you could also leave it.

Reply:
done